### PR TITLE
Adding skupack for arista 7124S switch

### PR DESCRIPTION
**Pull request by pscharla**
Adding skupack for the 7124S Arista switch.

@RackHD/corecommitters @zyoung51 @heckj @VulpesArtificem

### CODE REVIEW

File: arista-dcs-7124S/workflows/deploy-startup-config-graph.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            "taskDefinition": {
+                "friendlyName": "Deploy switch config",
+                "injectableName": "Task.Inline.Switch.Deploy.Arista",
+                "implementsTask": "Task.Base.Linux.Commands",
+                "options": {
+                    "startupConfig": null,
+                    "startupConfigUri": "{{ api.base }}/templates/{{ options.startupConfig }}",
+                    "commands": [
+                        {
+                            "downloadUrl": "/api/1.1/templates/arista-deploy-startup-config.py"
```
Comment (benbp) [inline arista-dcs-7124S/workflows/deploy-startup-config-graph.json]: Use `{{ startupConfigUri }}` template value here instead?

Comment (pscharla) [inline arista-dcs-7124S/workflows/deploy-startup-config-graph.json]: Yes - will change it.

Comment (pscharla) [inline arista-dcs-7124S/workflows/deploy-startup-config-graph.json]: Actually - the downloadUrl value is the path to the deploy script (arista-deplay-startup-config.py).  The startupConfigUri is the path of the startup-config script.  So I don't think I can do:

`"downloadUrl": "{{ startupConfigUri }}"`

I think that's what you were saying.  I think I could do:

`"downloadUrl": "{{ api.base }}/templates/arista-deploy-startup-config.py"`

if that is better.

Comment (benbp) [inline arista-dcs-7124S/workflows/deploy-startup-config-graph.json]: Oh, yeah I misread it. `{{ api.base }}` would still be good though, that way we can change the API version of the route internally.

File: arista-dcs-7124S/config.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Arista Switch",
+    "rules": [
+        {
+            "path": "version.Hardwaremodel",
+            "equals": "AristaDCS-7124S-F"
+        },
+        {
+            "path": "version.Internalbuildversion",
+            "equals": "4.13.7M-1877079.4137M.1"
```
Comment (yyscamper) [inline arista-dcs-7124S/config.json]: Why need the build version? Isn't the hardware model enough?

**Comment by yyscamper** [comment]
:+1: after clarify my question.

File: arista-dcs-7124S/config.json (hunk 2 above)
Comment (pscharla) [inline arista-dcs-7124S/config.json]: Was looking for unique identifiers, but you are right. Should be removed so we are not locked down to a specific build version.

Comment (pscharla) [inline arista-dcs-7124S/config.json]: By the way, this particular Arista switch is EOL but was the one I had available to test on.  Will need to procure other flavors of switches to test out the newer models.

File: arista-dcs-7124S/workflows/deploy-startup-config-graph.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+    "friendlyName": "Deploy Arista switch",
+    "injectableName": "Graph.Switch.Arista.Deploy",
+    "options": {
+        "deploy-config": {
```
Comment (benbp) [inline arista-dcs-7124S/workflows/deploy-startup-config-graph.json]: needs to match the "deploy-startup-config" label.

Comment (pscharla) [inline arista-dcs-7124S/workflows/deploy-startup-config-graph.json]: Right. Also changing it to "deploy-startup-config" in the config.json file. Thanks!

**Comment by benbp** [comment]
👍

**Pull request merged**